src/systems/filecoin_markets/storage_market/_index.md

```diff
@@ -3,14 +3,13 @@ menuTitle: Storage Market
-
-Storage Deal payments are processed at every successful PoSt submission and `StorageMarketActor` will move locked funds from `StorageClient` to `StorageProvider`. `SlashStorageDealCollateral` is also triggered on PoSt submission when a Sector containing a particular `StorageDeal` is faulty or miners fail to submit PoSt related to a `StorageDeal`. Note that `StorageProvider` does not need to be the same entity as the `StorageMinerActor` as long as the deal is stored on the Filecoin in at least one sector throughout the life time of the storage deal.
+Both `StorageClient` and `StorageProvider` need to first deposit Filecoin token into `StorageMarketActor` before participating in the storage market. `StorageClient` can then send a `StorageDealProposal` to the `StorageProvider` along with the data. A partially signed `StorageDeal` is called a `StorageDealProposal`. `StorageProvider` can then put this storage deal in their `Sector`, countersign the `StorageDealProposal` and result in a `StorageDeal`. A `StorageDeal` is only in effect when it is submitted to and accepted by the `StorageMarketActor` on chain before the `ProposalExpiryEpoch`. `StorageDeal` does not include a `StartEpoch` as it will come into effect at the block when the deal gets accepted into the network. Hence, `StorageProvider` should publish the deal as soon as possible.
 
+`StorageDeal` payments are processed at every successful PoSt submission and `StorageMarketActor` will move locked funds from `StorageClient` to `StorageProvider`. `SlashStorageDealCollateral` is also triggered on PoSt submission when a Sector containing a particular `StorageDeal` is faulty or miners fail to submit PoSt related to a `StorageDeal`. Note that `StorageProvider` does not need to be the same entity as the `StorageMinerActor` as long as the deal is stored in at least one `Sector` throughout the life time of the storage deal.
```

Comment:
Note: it may be that we end up processing `StorageDeal` payments at larger intervals than PoSt submissions. (TBD, for now this is fine)

Reply:
Yes, this has been added in CronAction that will be triggered by CronActor